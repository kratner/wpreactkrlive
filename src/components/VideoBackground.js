import React, { Component } from 'react';

class VideoBackground extends Component {
  render() {
    return (
      <div className="video-background">
        <video
          autoPlay
          loop
          className="video-background__video"
          muted
          playsInline
        >
          <source
            src="/img/20181215_154218.mp4"
            type="video/mp4"
            className="video-background__video__source"
          />
        </video>
      </div>
    );
  }
}

export default VideoBackground;
