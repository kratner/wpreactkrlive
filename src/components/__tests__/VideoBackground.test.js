import React from "react";
import { shallow } from "enzyme";
import VideoBackground from "../VideoBackground";

describe("VideoBackground", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<VideoBackground />);
    expect(wrapper).toMatchSnapshot();
  });
});
