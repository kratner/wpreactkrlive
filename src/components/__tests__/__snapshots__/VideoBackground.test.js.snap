// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoBackground renders correctly 1`] = `ShallowWrapper {}`;
